### ISSUE TITLE
pytorch to convert yolov4-tiny model and only get one output when infernce.

### ISSUE THREAD

**Issue by HuiJu1218** [Xilinx/Vitis-AI#1022]
Hi,
I had use vitis-ai 2.5 pytorch container to convert yolov4-tiny model. I had convert to xmodel successfully. But when I inference model on KV260 using python, I got only one output. I had print(len(output)) and I got len(output)=1. Can anyone have idea? [Here is my model](https://drive.google.com/drive/folders/1eE1oAnRI8ZULulv8X2c8P76SlgPFxR_P?usp=sharing)
![Selection_048](https://user-images.githubusercontent.com/55121392/191810930-a817fff5-b8d3-4953-99d5-acdd5c691de3.png)
![Selection_049](https://user-images.githubusercontent.com/55121392/191810959-6a5715a6-cb19-4608-ba69-1c8d26299e26.png)

**Comment by tuan3ihust** [comment]
Hello, did your compiled yolov4-tiny model has multiple DPU subgraph? 
I train yolov4-tiny with darknet and convert to caffe with Vitis AI 1.4 and after compiled, DPU subgraph number is 4
![image](https://user-images.githubusercontent.com/91899320/204116242-0628ed9e-d83a-4f2d-a360-5631e95272ff.png)

**Comment by huisunCompiler** [comment]
Hi @HuiJu1218 ,
Thanks for providing xmodels, which helps a lot. From python xir I find 2 outputs, could you please describe the problem more in detail. (which output is missing.) many thanks :)
```
>>> import xir
>>> xir.Graph.deserialize("compiled.xmodel")
<xir.Graph named 'YoloBody'>
>>> g = xir.Graph.deserialize("compiled.xmodel")
>>> g.get_tail_ops()
{<xir.Op named 'YoloBody__YoloBody_Sequential_yolo_headP4__Conv2d_1__4248_fix_'>, <xir.Op named 'YoloBody__YoloBody_Sequential_yolo_headP5__Conv2d_1__4167_fix_'>}
>>>
```

**Comment by huisunCompiler** [comment]
@tuanmonkeyy 
strided_slice-fix is not supported in vitis-ai 2.5 and 4 strided_slice-fix are partitioned to CPU. I recommend you update to vitis-ai release 3.0, the 4 strided_slice are optimized and the DPU subgraph is 1.
